### ISSUE TITLE
cannot set property nativevew #viewbase which has only getter undefined?

### ISSUE THREAD

**Issue by vijaykumarbollavaram** [manijak/nativescript-carousel#47]
Iam getting this on screen. What is the error

**Comment by manijak** [comment]
Make sure the plugin is installed properly (that it exists in the node_modules folder after build).

Version 3.x of the plugin requires TNS 3.x

**Comment by gchiocchio** [comment]
Plugin exists in directory, below the depencies; 

"dependencies": {
    "nativescript-carousel": "^3.1.0",
    "nativescript-theme-core": "~1.0.2",
    "tns-core-modules": "**3.2.0-2017-8-9-2**"
  },
  "devDependencies": {
    "awesome-typescript-loader": "~3.1.3",
    "babel-traverse": "6.4.5",
    "babel-types": "6.4.5",
    "babylon": "6.4.5",
    "copy-webpack-plugin": "~4.0.1",
    "extract-text-webpack-plugin": "~3.0.0",
    "lazy": "1.0.11",
    "nativescript-css-loader": "~0.26.0",
    "nativescript-dev-typescript": "^0.5.0",
    "nativescript-dev-webpack": "^0.7.3",
    "raw-loader": "~0.5.1",
    "resolve-url-loader": "~2.1.0",
    "typescript": "^2.4.2",
    "webpack": "~3.2.0",
    "webpack-bundle-analyzer": "^2.8.2",
    "webpack-sources": "~1.0.1"
  },

**Comment by manijak** [comment]
No, the Carousel is not tested with 3.2.x since it is not available yet. Latest stable is 3.1.1.
